test/rpc/test_data/get_events.py

```diff
@@ -9,82 +9,161 @@
+        }
+    },
+    "id": 0,
 }
 
 BLOCK_FROM_0_TO_LATEST_CHUNK_3_CONTINUATION_TOKEN = {
-    "from_block": "0",
-    "to_block": "latest",
-    "chunk_size": 3,
-    "continuation_token": "1",
+    "jsonrpc": "2.0",
+    "method": "starknet_getEvents",
+    "params": {
+        "filter": {
+            "from_block": "0",
+            "to_block": "latest",
+            "chunk_size": 3,
+            "continuation_token": "1",
+        }
+    },
+    "id": 0,
 }
 
 BLOCK_FROM_0_TO_1_CHUNK_3_CONTINUATION_TOKEN = {
-    "from_block": "0",
-    "to_block": "1",
-    "chunk_size": 3,
-    "continuation_token": "0",
+    "jsonrpc": "2.0",
+    "method": "starknet_getEvents",
+    "params": {
+        "filter": {
+            "from_block": "0",
+            "to_block": "1",
+            "chunk_size": 3,
+            "continuation_token": "0",
+        }
+    },
+    "id": 0,
 }
 
 BLOCK_FROM_0_TO_LATEST_WITH_ADDRESS = {
-    "from_block": "0",
-    "to_block": "latest",
-    "address": EXPECTED_FEE_TOKEN_ADDRESS,
-    "chunk_size": 10,
+    "jsonrpc": "2.0",
+    "method": "starknet_getEvents",
+    "params": {
+        "filter": {
+            "from_block": "0",
+            "to_block": "latest",
+            "address": EXPECTED_FEE_TOKEN_ADDRESS,
+            "chunk_size": 10,
+        }
+    },
+    "id": 0,
 }
 
 BLOCK_FROM_0_TO_LATEST_WITH_KEY = {
-    "from_block": "0",
-    "to_block": "latest",
-    "keys": ["0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9"],
-    "chunk_size": 10,
+    "jsonrpc": "2.0",
+    "method": "starknet_getEvents",
+    "params": {
+        "filter": {
+            "from_block": "0",
+            "to_block": "latest",
+            "keys": [
+                "0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9"
+            ],
+            "chunk_size": 10,
+        }
+    },
+    "id": 0,
 }
 
 BLOCK_FROM_0_TO_LATEST_WITH_KEYS = {
-    "from_block": "0",
-    "to_block": "latest",
-    "keys": [
-        "0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9",
-        "0x3db3da4221c078e78bd987e54e1cc24570d89a7002cefa33e548d6c72c73f9d",
-    ],
-    "chunk_size": 10,
+    "jsonrpc": "2.0",
+    "method": "starknet_getEvents",
+    "params": {
+        "filter": {
+            "from_block": "0",
+            "to_block": "latest",
+            "keys": [
+                "0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9",
+                "0x3db3da4221c078e78bd987e54e1cc24570d89a7002cefa33e548d6c72c73f9d",
+            ],
+            "chunk_size": 10,
+        }
+    },
+    "id": 0,
 }
 
 BLOCK_FROM_0_TO_LATEST_WITH_ADDRESS_AND_KEYS = {
-    "from_block": "0",
-    "to_block": "latest",
-    "address": EXPECTED_FEE_TOKEN_ADDRESS,
-    "keys": [
-        "0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9",
-        "0x3db3da4221c078e78bd987e54e1cc24570d89a7002cefa33e548d6c72c73f9d",
-    ],
-    "chunk_size": 10,
+    "jsonrpc": "2.0",
```

Comment:
All this `BLOCK_FROM_...` could be constructed using a common function which would insert properties like `jsonrpc`, `method`, `id` since the only thing different is the `filter` property of `params`. This refactor can be done in a separate PR.

Reply:
I was thinking about this but I remember that I was doing something similar 1 or 2 months ago and we decided that more clean way is not to use something like this.

I'll do it in this PR. It's faster than creating new task etc.